google-maps-api.html

```diff
@@ -103,11 +103,21 @@
 
       /** @private */
       libraryUrl: {
-        type: String,
-        computed: '_computeUrl(mapsUrl, version, apiKey, clientId, language, signedIn)'
+        type: String
       }
     },
 
+    observers: [
+      '_debounceComputeUrl(mapsUrl, version, apiKey, clientId, language, signedIn)'
+    ],
+
+    _debounceComputeUrl: function(mapsUrl, version, apiKey, clientId, language, signedIn) {
+      var _this = this;
+      _this.debounce('_computeUrl', function () {
+        _this.libraryUrl = _this._computeUrl(_this.mapsUrl, _this.version, _this.apiKey, _this.clientId, _this.language, _this.signedIn);
```

Comment:
debounce binds to `this`, so you dont need a separate var.

Each of the params to `computeUrl` should be the args to this method.